internal/rpchelp/helpdescs_en_US.go

```diff
@@ -24,6 +24,14 @@ var helpDescsEnUS = map[string]string{
 	"accountsyncaddressindex-branch":    "Number for the branch (0=external, 1=internal)",
 	"accountsyncaddressindex-index":     "The address index to synchronize to",
 
+	// AccountUnlockedCmd help.
+	"accountunlocked--synopsis": "Report account encryption and locked status",
+	"accountunlocked-account":   "String for the account",
+
+	// AccountUnlockedResult help.
+	"accountunlockedresult-encrypted": "If the account is encrypted (can be individually unlocked)",
```

Comment:
All accounts are encrypted, so let's say "Whether the account is individually encrypted with a separate passphrase".